src/index.tsx

```diff
@@ -62,23 +76,36 @@ createRoot(container).render(
   <StrictMode>
     <Provider store={store}>
       <FeatureFlagsProvider>
-        <QueryClientProvider client={queryClient}>
-          <HashRouter>
-            <LanguageProvider>
-              <Web3Provider>
-                <ApolloProvider client={apolloClient}>
-                  <BlockNumberProvider>
-                    <Updaters />
-                    <ThemeProvider>
-                      <ThemedGlobalStyle />
-                      <App />
-                    </ThemeProvider>
-                  </BlockNumberProvider>
-                </ApolloProvider>
-              </Web3Provider>
-            </LanguageProvider>
-          </HashRouter>
-        </QueryClientProvider>
+        <StatsigProvider
+          user={{
+            userID: getDeviceId(),
+            customIDs: { session_id: String(getSessionId() ?? -1) },
```

Comment:
what do `customIDs` represent here?

Reply:
i think they allow you to have a multi-ID for ID-based experiments. i was using this when i thought we wanted both User and Device IDs ... but i think we're only going to be using Device ID. i will remove